caiman_napari/algorithms/mcorr.py

```diff
@@ -65,5 +67,20 @@ def load_output(viewer, batch_item: pd.Series):
     MotionCorrectedMovie = np.reshape(Yr.T, [T] + list(dims), order='F')
     viewer.add_image(MotionCorrectedMovie)
 
+
+def load_projection(viewer, batch_item: pd.Series, proj_type):
+    path = batch_item['outputs'].item()["mcorr_output"][0]
+
+    Yr, dims, T = cm.load_memmap(path)
+    MotionCorrectedMovie = np.reshape(Yr.T, [T] + list(dims), order='F')
+
+    # Dict for Projections
+    choices = {'mean': np.mean(MotionCorrectedMovie, axis=0),
```

Comment:
This is going to calculate all of them beforehand which we don't want. You should do something like:

```python
MC_Projection = getattr(np, f"nan{proj_type}")(MotionCorrectedMovie, axis=0)
```